docs/core/tools/dotnet-aspnet-codegenerator.md

```diff
@@ -0,0 +1,193 @@
+---
+title: dotnet aspnet-codegenerator command
+description: The dotnet aspnet-codegenerator command scaffolds ASP.NET Core projects
+ms.date: 06/15/2019
+---
+# dotnet aspnet-codegenerator
+
+/aspnet/core/tutorials/first-mvc-app/adding-model
+
+[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
```

Comment:
First paragraph should be an introduction to the article for SEO, otherwise this include is the main SEO trigger which is bad.

Reply:
It is. I'm following the format of the other tools.

Reply:
Can you link me to the other tools? I don't know why but I have no idea which ones you're talking about 😄